features/correction/generator.py

```
@@ -164,17 +206,58 @@ def _validate(
+        raise ValueError(f"첨삭 설정값 검증에 실패했습니다: {exc}") from exc
+
+
+def _load_generator_fallback_max_retries() -> int:
     config_path = Path(__file__).resolve().parent / "config" / "generator.yaml"
 
     try:
         with config_path.open(encoding="utf-8") as file:
             config = yaml.safe_load(file) or {}
-        value = config.get("generator", {}).get("max_retries", _DEFAULT_MAX_RETRIES)
-        return int(value)
     except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)